mac

```diff
@@ -155,6 +154,11 @@ if brew list | grep --silent "qt@5.5"; then
   brew link --force qt@5.5
 fi
 
+if [ "$(which psql)" == "" ] ; then
+  fancy_echo "'psql' not found in PATH, attempting to add it..."
+  append_to_bash_profile 'export PATH="/usr/local/opt/postgresql@9.6/bin:$PATH"'
```

Comment:
This needs to be added to the PATH, but homebrew doesn't do this on installation. I've always had to do it manually on every laptop setup, so I instead automated it here.

I'm not sure if the installation path is always the same on every machine, so I'm not sure if this will work for everyone.

Reply:
Will this interact with other path settings, like RVM which says it has to be last?

Reply:
It shouldn't. If something needs to be last, it just needs to be put in the right place on the `bash_profile`, which is true anyways and the responsibility of the installer.